config/agencies.localdev.yml

```diff
@@ -1,119 +1,176 @@
 test:
```

Comment:
Note that this wasn't strictly required for this PR but will make things simpler when we add it to the schema.